api/datastore/sql/sql.go

```diff
@@ -82,13 +82,31 @@ var tables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 	app_id varchar(256) NOT NULL,
 	log text NOT NULL
 );`,
+
+	`CREATE TABLE IF NOT EXISTS fns (
```

Comment:
How come we've got both a migration and this? Seems like a duplication.

Reply:
sad fact of the way that migration works now - you need both and they have to describe the same target state.